core/domain/rules_registry_test.py

```diff
@@ -58,9 +58,9 @@ def test_get_html_field_types_to_rule_specs_for_previous_state_schema_version(
         with utils.open_file(spec_file_v41, 'r') as f:
             specs_from_json_v41 = json.loads(f.read())
 
-        self.assertDictEqual(
-            html_field_types_to_rule_specs_v41,
-            specs_from_json_v41)
+            self.assertDictEqual(
+                html_field_types_to_rule_specs_v41, specs_from_json_v41
+            )
 
     def test_get_html_field_types_to_rule_specs_for_unsaved_state_schema_version(  # pylint: disable=line-too-long
```

Comment:
If i'm right we don't cache here. Can we add that to the func name?

Reply:
Hi @gp201, I have made the required changes.